libs/windy-sounding/src/styles.less

```
@@ -131,424 +11,13 @@
-        width: 100%;
-        text-align: center;
-        white-space: nowrap;
-
-        &:before,
-        &:after {
-          display: inline-block;
-          position: relative;
-          top: -2px;
-        }
+// Skew-T and wind profile diagrams
+@import './style/skewt.less';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)